tests/main/interfaces-cups-control/task.yaml

```
@@ -55,7 +55,7 @@ execute: |
     echo "Then the snap command is able to print files"
     echo "Hello World" > "$TEST_FILE"
     test-snapd-cups-control-consumer.lpr "$TEST_FILE"
-    while "$(find "$HOME"/PDF -name '*.pdf' | wc -l)" -lt 1 ; do sleep 1; done
+    retry -n 30 --wait 1 sh -c "test \"$(find "$HOME"/PDF -name '*.pdf' | wc -l)\" -ge 1"
```

Keeps failing:
```
 retry: next attempt in 1.0 second(s) (attempt 25 of 30)
retry: command sh -c test "0" -ge 1 failed with code 1
retry: next attempt in 1.0 second(s) (attempt 26 of 30)
retry: command sh -c test "0" -ge 1 failed with code 1
retry: next attempt in 1.0 second(s) (attempt 27 of 30)
retry: command sh -c test "0" -ge 1 failed with code 1
retry: next attempt in 1.0 second(s) (attempt 28 of 30)
retry: command sh -c test "0" -ge 1 failed with code 1
retry: next attempt in 1.0 second(s) (attempt 29 of 30)
retry: command sh -c test "0" -ge 1 failed with code 1
retry: command sh -c test "0" -ge 1 keeps failing after 30 attempts
-----
.
```

this may be the same evaluation issue as I pointed out in another test. I think this should work (may require disabling a shellcheck rule):
```suggestion
    retry -n 30 --wait 1 sh -c 'test "$(find "$HOME"/PDF -name "*.pdf" | wc -l)" -ge 1'
```

---

I already fixed that. Couldn't use ' because it makes shellcheck unhappy: 
retry -n 30 --wait 1 sh -c 'test "$(find "$HOME"/PDF -name "*.pdf" | wc -l)" -ge 1'
^-- SC2148: Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.
                           ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.

---

may be writing garbage, but I belive that without the single quotes, assuming there is no *.pdf files in $HOME/PDF, the code at the call time expands to:
```
retry -n 30 --wait 1 sh -c "test \"0\" -ge 1"
```

---

ok, so I'll need to disable shellcheck to avoid errors, thanks for the suggestion!